src/pages/collection/series/TmdbLinking.tsx

```diff
@@ -216,6 +216,13 @@ const TmdbLinking = () => {
           [] as TmdbEpisodeXrefMappingRequestType[],
         );
 
+        // If we're not giving the server any clues about which series to link
+        // then we need to first create the auto links before sending the
+        // mappings.
+        if (isNewLink && every(newMappings, link => link.TmdbID === 0)) {
+          await createAutoLinks({ tmdbShowID: tmdbId });
+        }
+
```

Comment:
you could refactor the above if-else and this to have a single `createAutoLinks` line

Reply:
I'll move this change (after fixing it) and the above to their own PR.